.github/workflows/ci.yml

```
@@ -196,7 +198,7 @@ jobs:
             APP=${{ matrix.component}}
             VERSION=${{ steps.info.outputs.version }}
           pull: true
-          outputs: type=docker,dest=image.tar
+          outputs: type=docker,dest=image.targl
```

I believe this is an unintended change.

```suggestion
          outputs: type=docker,dest=image.tar
```

---

I also think it's typo, somehow it got me 👀